### ISSUE TITLE
Error 400: Request must contain a valid upload token.

### ISSUE THREAD

**Issue by phoenix1887** [gphotosuploader/gphotos-uploader-cli#132]
Uploaded a bunch of images successfully. Now it's just giving these errors repeatedly when running it and then quitting. What's going on? How to fix this?

2019/09/19 11:08:12 Looking up upload URLs database for  /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00154.ARW
2019/09/19 11:08:12 Uploading /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00154.ARW
2019/09/19 11:08:12 Checking status of upload URL 'https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2Up8FcRcIDQZc-N7gbyQMQ54rWEraPAgeoHTIl-3cWODQqrCYG2_89QHpbEoTIcU1Bfav_oPm-rWHT49SrWbjzcy8cpZFA&upload_protocol=resumable'
2019/09/19 11:08:13 Status of upload URL 'https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2Up8FcRcIDQZc-N7gbyQMQ54rWEraPAgeoHTIl-3cWODQqrCYG2_89QHpbEoTIcU1Bfav_oPm-rWHT49SrWbjzcy8cpZFA&upload_protocol=resumable' is 'final'
2019/09/19 11:08:13 Getting new upload URL for '/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00154.ARW'
gphotos-api2019/09/19 11:08:13 Upload completed: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00154.ARW
2019/09/19 11:08:14 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:15 Uploading object: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00155.ARW
2019/09/19 11:08:15 Looking up upload URLs database for  /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00155.ARW
2019/09/19 11:08:15 Uploading /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00155.ARW
2019/09/19 11:08:15 Getting new upload URL for '/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00155.ARW'
gphotos-api2019/09/19 11:08:15 Upload completed: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00155.ARW
2019/09/19 11:08:17 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:17 Uploading object: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00156.ARW
2019/09/19 11:08:17 Looking up upload URLs database for  /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00156.ARW
2019/09/19 11:08:17 Uploading /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00156.ARW
2019/09/19 11:08:17 Getting new upload URL for '/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00156.ARW'
gphotos-api2019/09/19 11:08:17 Upload completed: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00156.ARW
2019/09/19 11:08:18 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:19 Uploading object: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00157.ARW
2019/09/19 11:08:19 Looking up upload URLs database for  /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00157.ARW
2019/09/19 11:08:19 Uploading /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00157.ARW
2019/09/19 11:08:19 Getting new upload URL for '/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00157.ARW'
gphotos-api2019/09/19 11:08:19 Upload completed: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00157.ARW
2019/09/19 11:08:20 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:20 Uploading object: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00158.ARW
2019/09/19 11:08:20 Looking up upload URLs database for  /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00158.ARW
2019/09/19 11:08:20 Uploading /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00158.ARW
2019/09/19 11:08:20 Getting new upload URL for '/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00158.ARW'
gphotos-api2019/09/19 11:08:21 Upload completed: file=/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00158.ARW
2019/09/19 11:08:21 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:24 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:27 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:29 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:30 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:32 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:35 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:38 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:40 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:41 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:42 Error during upload, sleeping for 10 seconds before retrying...
2019/09/19 11:08:45 Error uploading file '/media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00154.ARW', storing upload URL 'https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2Uq4WUbvCfXaIvgbopFANKnVWdBQ0vjihn6ATNKt-xbxn636QqahEhAbyFCvOv0H8s5gCsyXtC64Lyv6Un3XBJia_9e4QDRAmviNmDQtky4JKsTHhyk&upload_protocol=resumable'
2019/09/19 11:08:45 Stored upload URL for file /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00154.ARW (https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2Uq4WUbvCfXaIvgbopFANKnVWdBQ0vjihn6ATNKt-xbxn636QqahEhAbyFCvOv0H8s5gCsyXtC64Lyv6Un3XBJia_9e4QDRAmviNmDQtky4JKsTHhyk&upload_protocol=resumable)
2019/09/19 11:08:45 failed uploading image: failed uploading image: failed adding media /media/mattimus/HD/Photo Shoots/Sava Yoga/DSC00154.ARW
 --- at /home/mattimus/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go@v1.0.6/lib-gphotos/uploads.go:278 (Client.UploadFileResumable) ---
Caused by: googleapi: Error 400: Request must contain a valid upload token., badRequest

Latest version, just installed a few hours ago, Ubuntu 18.04 KDE Plasma
gphotos-uploader-cli v0.8.3 (build: 55c6363)

**Comment by phoenix1887** [comment]
Also I should add that I have multiple other folder objects listed in the "jobs" array in the configuration file, it's not continuing to the next folders to upload those files as you would expect, it's just quitting-- but perhaps this is part of the other open issues with the program quitting when it hits errors rather than continue on to the next images. This is pretty important!!

**Comment by pacoorozco** [comment]
Duplicated #110 It's the same bug :-)

**Comment by eng3** [comment]
I am just starting.  I already have 99% of my library on google photos (back in the day with Picasa and through manual album creation).  I tried letting the upload go on its own (failing every file) thinking the upload database will eventually become up to date and any new albums should get created moving forward.  I stopped gphotos because I reached my upload limit.  Upon restarting, I now get "uploads.db, err=resource temporarily unavailable".  I guess the uploads.db got corrupted?  What is the recommended way to get around this issue and the general issue of already having a library on google photos.  I'd prefer not to completely delete all my uploaded albums since I'm not 100% sure this tool even works for me.
